src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/xlf/PropertyPageResources.es.xlf

```diff
@@ -49,7 +49,7 @@
       </trans-unit>
       <trans-unit id="FindBtn">
         <source>Find...</source>
-        <target state="new">Find...</target>
+        <target state="translated">Buscar:</target>
```

Comment:
Small question: is there a reason why it's translated to `:` instead of `...`?

Reply:
Good catch! I wonder what other buttons have on them

Reply:
thank you for this feedback. Issue fixed